SFML.NuGet.props

```diff
@@ -5,7 +5,7 @@
   <PropertyGroup>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
 
-    <Version>2.5.0</Version>
+    <Version>2.5.1</Version>
```

Comment:
I wouldn't increase the version now. Versioning as well as release tagging should be done once a new version is ready for release.